frugalos_segment/src/client/mds.rs

```diff
@@ -164,9 +171,9 @@ impl MdsClient {
     ) -> impl Future<Item = (ObjectVersion, bool), Error = Error> {
         debug!(self.logger, "Starts PUT: id={:?}", id);
         let put_content_timeout = Seconds(if let Deadline::Within(d) = deadline {
-            d.as_secs() + 60 // XXX: hard-coding
+            d.as_secs() + self.client_config.minimum_put_content_timeout.0
```

Comment:
@sile I want to change this specification to `max(deadline, minimum_put_content_timeout)`. Is there any problem?

Reply:
I think the original code intend to ensure that timeout in seconds is always greater than 60secs.

Reply:
I understood.